source/Octopus.Tentacle/Communications/NamespaceMappingSerializationBinderDecorator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Newtonsoft.Json.Serialization;
+
+namespace Octopus.Tentacle.Communications
+{
+    public class NamespaceMappingSerializationBinderDecorator : ISerializationBinder
```

Comment:
This is important - maps from the old namespace (Octopus.Shared.Contracts) to the new namespace (Octopus.Tentacle.Contracts).